src/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ install(
 install(
 	EXPORT "${PROJECT_NAME}-targets"
 	NAMESPACE "${PROJECT_NAME}::"
-	DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
+	DESTINATION "cmake"
```

Comment:
This install path is on cmake's standard search paths, and helps to avoid conflicts with similarly named files. Please do not change it without good reason.

Reply:
Would you be fine with ``cmake/${PROJECT_NAME}``?
You may be right, but I usually see a root cmake folder

Reply:
Otherwise, I can change the portfile

Reply:
This pattern is very common:
https://github.com/martinmoene/span-lite/blob/master/CMakeLists.txt#L117
https://github.com/catchorg/Catch2/blob/devel/CMakeLists.txt#L50
https://github.com/gabime/spdlog/blob/v1.x/CMakeLists.txt#L275